tests/end2end/data/hints/link_inject.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>A link to use hints on</title>
+        <script>
+            function injectPort() {
+                const queryString = document.location.search;
+                const params = new URLSearchParams(queryString);
```

Comment:
Webkit says:
>   12:37:26.409 DEBUG    js         shared:javascript_log_message:190 [file:///__w/qutebrowser/qutebrowser/tests/end2end/data/hints/link_inject.html?port=46851:9] ReferenceError: Can't find variable: URLSearchParams
>   12:37:26.448 DEBUG    webelem    webelem:click:407 Clicking <qutebrowser.browser.webkit.webkitelem.WebKitElement html='<a href="http://localhost:&lt;port&gt;/data/hello.txt" id="link">Follow me!</a>'> with click_target ClickTarget.normal, force_event False

Reply:
Bummer, looks like webkit does not support [`URLSearchParams`](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams#browser_compatibility) then..

We could work around that by doing the parameter parsing manually.

Reply:
If you do `#port` and read the fragment from JS, there might be no parsing necessary. But I'm also fine with just `@qtwebkit_skip`, even more so because this is a QtWebEngine-specific workaround.

Reply:
I'm more than happy to take `@qtwebkit_skip` then!